spec/delta_spec.rb

```diff
@@ -42,7 +42,7 @@
     it 'passes options on to sets' do
       opts = { decay: false }
       mock_set = double()
-      mock_set.should_receive(:fetch).with(opts) { [] }
+      expect(mock_set).to receive(:fetch).with(opts) { [] }
```

Comment:
this syntax isn't supported in `rspec 2.13.0`, though the test itself passes with the old syntax. will clean up on master.